### DISCUSSION TITLE
SOLUTION: Unable to deploy SPFx webpart in Sharepoint 2019 onprem

### DISCUSSION THREAD

**Discussion by mhazebroucq** [SharePoint/sp-dev-docs#7887] [Show and tell]
Hello everybody,

I was unable to deploy an SPFx development on a SharePoint 2019 platform.
I searched for several months without finding answers.

But I recently came across an article on the Quest support site that helped me resolve this issue:
[https://support.quest.com/fr-fr/metalogix-storagepoint/kb/331962/can-t-activate-storagepoint-modern-ui-spfx-extension-in-sharepoint-2019](https://support.quest.com/fr-fr/metalogix-storagepoint/kb/331962/can-t-activate-storagepoint-modern-ui-spfx-extension-in-sharepoint-2019)

I'm posting it here if it can help anyone

#6438

**Comment by ghost** [comment]
Thank you for reporting this issue. We will be triaging your incoming issue as soon as possible.
